File: ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/SdsStreamViewProperty.cs
// <copyright file="SdsStreamViewProperty.cs" company="OSIsoft, LLC">
//
// </copyright>

using System;
using System.Collections.Generic;
using System.Text;

namespace SdsRestApiCore
{
    public class SdsStreamViewProperty
    {
        public string SourceId
        {
            get;
            set;
        }

        public string TargetId
        {
            get;
            set;
        }

        public SdsStreamView SdsStreamView
        {
            get;
            set;
        }
    }
}

File: ocs_samples/library_samples/Java/ocs_sample_library_preview/src/main/java/com/github/osisoft/ocs_sample_library_preview/sds/SdsStreamViewProperty.java
/** SdsStreamViewProperty.java
 * 
 */

package  com.github.osisoft.ocs_sample_library_preview.sds;

/** 
 * SdsStreamViewProperty
 */
public class SdsStreamViewProperty {
    private String SourceId;
    private String TargetId;
    private SdsStreamView SdsStreamView;

    /**
     * gets SourceId
     * @return
     */
    public String getSourceId() {
        return SourceId;
    }

    /**
     * sets SourceId
     * @param SourceId
     */
    public void setSourceId(String SourceId) {
        this.SourceId = SourceId;
    }

    /**
     * gets targetid
     * @return
     */
    public String getTargetId() {
        return TargetId;
    }

    /**
     * sets targetid
     * @param targetId
     */
    public void setTargetId(String targetId) {
        this.TargetId = targetId;
    }

    /**
     * gets SdsStreamView
     * @return
     */
    public SdsStreamView getSdsStreamView() {
        return SdsStreamView;
    }

    /**
     * sets SdsStreamView
     * @param sdsStreamView
     */
    public void setSdsStreamView(SdsStreamView sdsStreamView) {
        this.SdsStreamView = sdsStreamView;
    }
}
